client/db.go

```diff
@@ -196,6 +196,14 @@ type Store interface {
 		transform immutable.Option[model.Lens],
 	) ([]CollectionDefinition, error)
 
+	// RefreshViews refreshes the caches of all views matching the given options.  If no options are set all views
```

Comment:
suggestion: `If no options are set,`

Reply:
Leaving this one as is, I don't think the comma helps, and I think I'm usually guilty of over-using them

Reply:
Just adding that without the comma it reads `If no options are set-all-views...`

Reply:
Discussed on discord, adding comma

- [ ] add comma